RELEASING.md

```diff
@@ -106,16 +113,85 @@ done
 
 Check your e-mail and make sure the number of “you’ve published this module” emails matches the number you expect.
 
-## Publish the GitHub Release
+### Publish the GitHub Release
 
 Publish the GitHub release, ensuring that the tag points to the newly landed commit corresponding to release proposal `x.y.z`.
 
-## Update CHANGELOG
+### Update CHANGELOG
 
 - After releasing is done, update the [CHANGELOG.md](https://github.com/open-telemetry/opentelemetry-js/blob/master/CHANGELOG.md) and start new Unreleased label.
 - Create a new commit with the exact title: `Post Release: update CHANGELOG.md`.
 - Go through PR review and merge it to GitHub master branch.
 
-## Known Issues
+### Known Issues
 
 - The `examples/` and `getting-started/` folders are not part of lerna packages, we need to manually bump the version in `package.json`.
+
+## Versioning
```

Comment:
overall i'm fine (plus the modification discussed about the SIG), could we add a link to this inside the readme ?

Reply:
+1, remove experiential, use version 0, new package for new (significant?) experimental changes, don't rev the API version for every change (unless the API explicitly changes)

Reply:
>link to this inside the readme

yes

>plus the modification discussed about the SIG

[`114d83c` (#1737)](https://github.com/open-telemetry/opentelemetry-js/pull/1737/commits/114d83cd0d955d5ee3d19c9283810e5d06f0b12a)